map_gui/src/tools/updater.rs

```diff
@@ -180,7 +88,7 @@ pub fn prompt_to_download_missing_data<A: AppLike + 'static>(
                     };
                     Transition::Replace(PopupMsg::new(
                         ctx,
-                        "Download complete. Please try again",
+                        "Download complete. Try again!",
```

Comment:
After the download succeeds, rather than prompting them to click the button again, it seems preferable to "click it for them" and transition them directly to loading at this point.

Is there a reason we intentionally don't do that? Or is it that it's prohibitively non-trivial to implement?

Reply:
See the comment on city_picker, "Both the code and UX of this aren't great..."

It's hard. :(